lib/config/presets/internal/replacements.ts

```diff
@@ -146,3 +133,38 @@ export const presets: Record<string, Preset> = {
     ],
   },
 };
+
+const mui = [
+  ['unstyled', 'core'],
+  ['icons', 'icons-material'],
+  ['core', 'material'],
+  ['system'],
+  ['styles'],
+  ['lab'],
+  ['types'],
+  ['styled-engine'],
+  ['styled-engine-sc'],
+  ['private-theming'],
+  ['codemod'],
+];
+
+for (const pack of mui) {
```

Comment:
I like the concept, but I wonder if it would be useful to make it a little more general even at this stage. #14367 has similar changes that span a lot of similarly named packages, and a general version of this would be useful.